### ISSUE TITLE
Unify markdown packages

### ISSUE THREAD

**Issue by elboletaire** [vocdoni/ui-scaffold#784]
Right now we're using two different package sets for creating and rendering markdown files. The one used to create markdown is `lexical`, which is actually a framework for WYSIWYG editors, so we should think about reusing that for rendering them too (rather than using the currently installed package `react-markdown`, which does not use lexical at all).

**Comment by elboletaire** [comment]
Not really. [We're still using react-markdown](https://github.com/vocdoni/ui-components/blob/main/packages/chakra-components/src/components/layout/Markdown.tsx) for the rendering 👀

**Comment by AlexanderArce** [comment]
> Not really. [We're still using react-markdown](https://github.com/vocdoni/ui-components/blob/main/packages/chakra-components/src/components/layout/Markdown.tsx?rgh-link-date=2025-08-25T13%3A39%3A07.000Z) for the rendering 👀

Then maybe it's an issue for ui-components not for ui-scaffold?

**Comment by elboletaire** [comment]
Certainly, though I'll have to change the name in there.
